File: Python/Utils/DesiredCapabilities.py

def GetAndroidWebDesiredCapability():
    capability = {}
    capability['sessionName'] = '[Python] Android Web'
    capability['sessionDescription'] = 'Web automation on android device using Python'
    capability['browserName'] = 'chrome'
    capability['platformName'] = 'Android'
    capability['deviceName'] = ''
    capability['platformVersion'] = ''
    capability['udid'] = ''
    return capability

def GetAndroidAppDesiredCapability():
    capability = {}
    capability['sessionName'] = '[Python] Android App'
    capability['sessionDescription'] = 'App automation on android device using Python'
    capability['app'] = 'https://s3-ap-southeast-1.amazonaws.com/kobiton-devvn/apps-test/ApiDemos-debug.apk'
    capability['platformName'] = 'Android'
    capability['deviceName'] = ''
    capability['platformVersion'] = ''
    capability['udid'] = ''
    return capability

File: Ruby/Utils/DesiredCapabilities.rb
def getAndroidWebDesiredCapability
  return caps = {
    sessionName:         '[Ruby] Android web automation',
    sessionDescription:  'Testing android web using Ruby',
    browserName:         'chrome',
    platformName:        'Android',
    deviceName:          '', # <-- change here
    platformVersion:     '', # <-- change here
    udid:                ''  # <-- change here
  }
end

def getAndroidAppDesiredCapability
  return caps = {
    sessionName:        '[Ruby] Android app automation',
    sessionDescription: 'Testing android app using Ruby',
    app:                'https://s3-ap-southeast-1.amazonaws.com/kobiton-devvn/apps-test/ApiDemos-debug.apk',
    platformName:       'Android',
    deviceName:         '', # <-- change here
    platformVersion:    '', # <-- change here
    udid:               ''  # <-- change here
  }
end
